BUILD

```diff
@@ -13,3 +13,10 @@ label_flag(
     build_setting_default = "//:.mypy.ini",
     visibility = ["//visibility:public"],
 )
+
+label_flag(
+    name = "mypy",
+    # No usable default. Force users to inject mypy.
+    build_setting_default = "//tools:empty_library",
```

Comment:
I would recommend not doing so. The reason is, that using a default mypy requires fetching it via rules_python and pip_parse, which makes using this project via a WORKSPACE file quite clunky. Essentially this would require calling the multiple steps to load mypy with some names hard coded in this project.
On the other hand, most likely only Python users will use this project at all, so adding mypy to their deps and setting the label_flag are literally 2 lines of work.